README.md

```diff
@@ -7,10 +7,34 @@ of files that need to be read inside of a container.
 
 ## Usage
 
-This is a basic `docker-compose.yml` that demonstrates usage of this container.
+These are basic `docker-compose.yml` configurations that demonstrates usage of this container.
+
+### Version 3
 
 ```
+version: '3'
 
+services:
+  web:
+    image: php:7.0-apache
+    volumes:
+      - /var/www/myapp
+
+  bg-sync:
+    image: cweagans/bg-sync
+    volumes:
+      - .:/source
+      - /var/www/myapp
```

Comment:
This doesn't seem to work... it adds a volume in this path in bg-sync, but it doesn't do anything to share those files over to the `web` container. I just tried on my v3 Docker Compose file and I can see /source contents in the bg-sync container, but nothing is shared into my web container.

Reply:
See: https://github.com/cweagans/docker-bg-sync/issues/23#issuecomment-597959059

Reply:
Thanks @geerlingguy :)

Instead of chaning my merge request I'm closing because it's wrong, I already deleted my clone of this repo and cweagans didn't reply